internal/cli/atlas/security/customercerts/create.go

```diff
@@ -26,6 +26,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const createTemplate = `CAS
+{{.Cas}}
+`
+
```

Comment:
[q] What did this look like without the template?

I'm looking at the output fro `gpg` and wondering should it just be the key, so you can then pipe the output to file and use it directly somewhere. Right now you'd need to strip the `CAS` line at the start.

Reply:
@robcarlan-mlab I'm with @colm-quinn in that I feel like the "human" output could be just straight up the certificate, no title, but that for the describe, for the create I think we should follow other commands with just a "success message"

@colm-quinn for context in this command (create) the user is providing the certificate so returning an answer with the same certificate the user provided is kind of a meh for me

Reply:
> @colm-quinn for context in this command (create) the user is providing the certificate so returning an answer with the same certificate the user provided is kind of a meh for me

Got it and yes agreed.

Reply:
Yeah that makes sense to me too 👍